index.html

```diff
@@ -30,6 +30,130 @@ <h1>Frehiywot <br />Nega</h1>
     <br>
-    <div class="image_geometry_2"></div> 
+    <div class="image_geometry_2"></div>
+    <h2>Projects</h2>
+  </section> 
+
+  <section class="Works-section" >
+   
+      <div class="worksection1"> </div>
+      <div class="rectangle">
+        <h3>Project name goes here</h3>
+        <ul class="buttons">
+          <li><button class="btn HTMLCSS">HTML/CSS</button></li>
+          <li><button class="btn Ruby">Ruby on Rails</button></li>
+          <li><button class="btn Js ">JavaScript</button></li>
+          </ul>
+          <button class="see_projectbtn">
+            <span>See this project</span>
+            <img src="image/Union.png" alt="arrow"  />
+          </button>
+        </div>
+
+        <div class="worksection2"> </div>
+      <div class="rectangle">
+        <h3>Project name goes here</h3>
+        <ul class="buttons">
+          <li><button class="btn HTMLCSS">HTML/CSS</button></li>
+          <li><button class="btn Ruby">Ruby on Rails</button></li>
+          <li><button class="btn Js ">JavaScript</button></li>
+          </ul>
+          <button class="see_projectbtn">
+            <span>See this project</span>
+            <img src="image/Union.png" alt="arrow"  />
+          </button>
+        </div>
+
+        <div class="worksection3"> </div>
+      <div class="rectangle">
+        <h3>Project name goes here</h3>
+        <ul class="buttons">
+          <li><button class="btn HTMLCSS">HTML/CSS</button></li>
+          <li><button class="btn Ruby">Ruby on Rails</button></li>
+          <li><button class="btn Js ">JavaScript</button></li>
+          </ul>
+          <button class="see_projectbtn">
+            <span>See this project</span>
+            <img src="image/Union.png" alt="arrow"  />
+          </button>
+        </div>
+
+        <div class="worksection4"> </div>
+      <div class="rectangle">
+        <h3>Project name goes here</h3>
+        <ul class="buttons">
+          <li><button class="btn HTMLCSS">HTML/CSS</button></li>
+          <li><button class="btn Ruby">Ruby on Rails</button></li>
+          <li><button class="btn Js ">JavaScript</button></li>
+          </ul>
+          <button class="see_projectbtn">
+            <span>See this project</span>
+            <img src="image/Union.png" alt="arrow"  />
+          </button>
+        </div>
+
+        <div class="worksection5"> </div>
+      <div class="rectangle">
+        <h3>Project name goes here</h3>
+        <ul class="buttons">
+          <li><button class="btn HTMLCSS">HTML/CSS</button></li>
+          <li><button class="btn Ruby">Ruby on Rails</button></li>
+          <li><button class="btn Js ">JavaScript</button></li>
+          </ul>
+          <button class="see_projectbtn">
+            <span>See this project</span>
+            <img src="image/Union.png" alt="arrow"  />
+          </button>
+        </div>
+        
+  </section>
+  <section class="About-section">
```

Comment:
- [ ] Your margins are not on point compared to the given Figma template. Please fix the spacing.
- [ ] You can reset `margin` and `padding` at the start of the CSS by:
```
* {
  margin: 0;
  padding: 0;
}
```

![image](https://user-images.githubusercontent.com/92215878/191581828-16dc0012-3845-457b-a477-2025f42db976.png)